indexer/src/main/java/bio/terra/tanagra/indexing/job/ComputeEntityLevelDisplayHints.java

```diff
@@ -150,6 +140,9 @@ public void run(boolean isDryRun) {
               }
             });
 
+    // Poll for table existence before we try to insert the hint rows. Maximum 60 x 10 sec = 10 min.
+    pollForTableExistenceOrThrow(getAuxiliaryTable(), 60, Duration.ofSeconds(10));
```

Comment:
Given that we previously expected the table to appear with 5s, waiting 10m seems a bit excessive and just makes failures slower. A minute or two would be plenty.